### PR TITLE
[SRM-512] SDPAP-6730 Remove deprecated "drupal_get_path()"

### PR DESCRIPTION
**Pull request by mayurngondhkar**
**Jira:**
https://digital-vic.atlassian.net/browse/SRM-512
https://digital-vic.atlassian.net/browse/SDPAP-6730

**Changes**

1. Remove deprecated `drupal_get_path()` function

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "drupal/crop": "^2.1",
-        "dpc-sdp/tide_core": "^3.0.1",
+        "dpc-sdp/tide_core": "dev-feature/SRM-512_update_drupal_to_9.4",
```
Comment (MdNadimHossain) [inline composer.json:8]: Revert it back after tide_core release and before merging this PR.

File: dev-tools.sh [hunk 2]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=960eba39ed9543b17654dbd5e06f3a2ab2b1fb4b
```
Comment (MdNadimHossain) [inline dev-tools.sh:36]: Revert it back after dev_tools release and before merging this PR.

**Review by MdNadimHossain — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (mayurngondhkar) [reply composer.json]: Changes reverted

File: dev-tools.sh (hunk 2 above)
Comment (mayurngondhkar) [reply dev-tools.sh]: Changes reverted

**Pull request merged**